### DISCUSSION TITLE
Notion.so add hints for items with inline styles that include "cursor: pointer"

### DISCUSSION THREAD

**Discussion by christianmagill** [gdh1995/vimium-c#910] [Q&A] [answered]
There are a lot of elements in Notion that have clickable events but are not coded as buttons or links. Most have inline styles that include "cursor: pointer" but no classes to target.

I've been attempting to target using some variation of the selector `[style*="cursor: pointer"]`, but I'm not having any luck.

My latest attempt looks like this...

`map f LinkHints.activate clickable="[style*=\"cursor:\u0020pointer\"]"`

Any help would be much appreciated.

**Comment by gdh1995** [comment]
No, CSS selectors can not select "all `cursor: pointer` elements", because there's no way to select those elements which match a `cursor: pointer` rule in a CSS file or a `<style>` node.

Vimium C won't read the style property during detecting clickable elements, mainly because it's a bit too expensive to check it.

**Comment by gdh1995** [comment, marked as answer]
Oh I didn't read your question carefully, sorry.

During my further tests, most buttons on Notion.so have been hinted as expected. So I'm not sure what elements did you mean.

You selector `"[style*=\"cursor:\u0020pointer\"]"` looks correct. Maybe you can try `map f LinkHints.activate clickable="[style*=pointer]"` if the first does fail

**Reply by christianmagill** [reply]
I wanted to hint elements like table cells in databases. Your suggested code worked. Thanks!
